src/tools/cephfs/shell/cephfs-shell

```diff
@@ -15,14 +15,18 @@ import re
 import shlex
 import stat
 import errno
+import distro
 
 from cmd2 import Cmd
 from cmd2 import __version__ as cmd2_version
 from packaging.version import Version
 
 # XXX: In cmd2 versions < 1.0.1, we'll get SystemExit(2) instead of
 # Cmd2ArgparseError
-if Version(cmd2_version) >= Version("1.0.1"):
+DFLAG = False
```

Comment:
Please comment the usage of this flag and maybe its better to call it an override.